content/markdown/01-basic-r/01-06-workflow.rmd

```diff
@@ -0,0 +1,284 @@
+---
+output:
+  pdf_document: default
+  html_document: default
+---
+# Workflow in R
+**Author:** Cole Brookson
+**Date:** 02 July 2022
+
+This section will take you through a set of best practices for optimizing your daily workflow in R. **NOTE:** This will progress from beginner topics to more advanced usage, so please only operationalize from this section what is relevant to you. 
```

Comment:
This is a good intro, but we haven't gone through what comes next yet in the manual - should this go somewhere different?